includes/definitions/arista_mos.yaml

```diff
@@ -0,0 +1,24 @@
+os: arista_mos
+text: 'Arista MOS'
+type: network
+icon: arista
+group: arista
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
+    - { graph: device_processor, text: 'CPU Usage' }
+    - { graph: device_mempool, text: 'Memory Usage' }
+discovery:
+    -
+        sysObjectID:
+            - .1.3.6.1.4.1.43191.1.6.7
+poller_modules:
+    wireless: false
```

Comment:
Hi @hanserasmus 
Could you please avoid disabling modules, unless they fail on your device.